types/svgo/index.d.ts

```diff
@@ -29,7 +29,10 @@ export type AddAttributesToSVGElementPlugin = DefaultPlugin<
 /**
  * adds classnames to an outer <svg> element
  */
-export type AddClassesToSVGElementPlugin = DefaultPlugin<'addClassesToSVGElement'>;
+export type AddClassesToSVGElementPlugin = DefaultPlugin<
+    'addClassesToSVGElement',
+    { className?: string | undefined; classNames?: string[] | undefined }
```

Comment:
are we sure 'undefined' is allowed here? It can result in an error, strict ooptional property support can do harm here:
https://github.com/svg/svgo/blob/main/plugins/addClassesToSVGElement.js#L60-L66
(there will be an error at runtime, if undefined is provided IMO)

Reply:
Thanks for the quick review - and catching this. You are correct 🙂 Fixed

Reply:
Reading the error message
```ts
var ENOCLS = `Error in plugin "addClassesToSVGElement": absent parameters.
It should have a list of classes in "classNames" or one "className".
```

could we even do
```ts
{ className: string; } | { classNames: string[] }
```
to force exactly one of the parameters?

Reply:
Good idea. I did this via `{ className?: string; classNames?: never; } | { className?: never; classNames?: string[]; }`

Reply:
Is there a reason to have the question marks there which I don't see?
`{}` is assignable `{ className?: string; classNames?: never; } | { className?: never; classNames?: string[]; }`
as opposed to
`{ className: string; classNames?: never; } | { className?: never; classNames: string[]; }`
which is what I would have written myself.

Reply:
No reason - removed the extra `?`! 👍  @petrzjunior